build/packaging/linux/bin/kend

```diff
@@ -203,6 +198,68 @@ start() {
         OPTIONS="$OPTIONS --autorestart.enable"
     fi
 
+    if [[ ! -z $KES_DB_TYPE ]] && [[ $KES_DB_TYPE != "" ]]; then
```

Comment:
@whoisxx Do you needs `KES` prefix? Most of the flags can be used in ordinary ENs except for the flags string with `--kes.`. It also has a different naming convention with other flags.

Reply:
@aidan-kwon I added `KES` prefix for KES users to easily setup configuration. if it could be used by EN users, It could be better to change. Do you recommend something likes `DB_TYPE`?

Reply:
@whoisxx Yes. I think it's okay if only the prefix is removed.

Reply:
remove KES prefix, only left `KES_NODETYPE_SERVICE_ENABLE` for kes.nodetype.service flag